Installing this repository via NPM fails: premature close

### Description

Attempting to install this repository directly via NPM fails due to invalid JSON (a trailing comma).

### Steps to Reproduce

1. Attempt to install: `npm i github:playcanvas/playcanvas-tween`
2. Error produced: `npm ERR! premature close`

--- a/package.json
+++ b/package.json
@@ -17,5 +17,5 @@
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/playcanvas/playcanvas-tween/issues"
-  },
+  }
 }

